packages/plugin-ext/src/hosted/node/hosted-plugin.ts

```diff
@@ -87,6 +87,10 @@ export class HostedPluginSupport {
         }
     }
 
+    public killProcessTree(parentPID: number): void {
```

Comment:
Hi @marechal-p , thanks for review.

1. I removed public modifier although:
a) some methods were marked as public already there and entire project is not consistent in that manner. 
b) I personally prefer the explicit way as it also conforms to existing private/protected modifiers.

2) I also removed public from existing methods - for consistency. and did a little sorting- moving private method to bottom.

3) About the middleman suggestion, IMO this should stay as it is as HostedInstanceManager has already HostedPluginSupport injected to it and HostedPluginSupport has HostedPluginProcess already injected to it - so I don't see advantage of injected HostedPluginProcess also to HostedInstanceManager - but if you think otherwise then I'll change it.

Thanks

Reply:
1) The project could be more consistent indeed, but there's a clear incline towards omitting the `public` keyword.
2) Thanks!
3) I don't see an issue with two components depending on the same one, if it avoids polluting the public API of one with API from the other, if that makes sense. I think that would be for the best API wise.

Reply:
@marechal-p 
3. removed killprocesstree from hostedpluginsupport.

Please re-review PR. 
Thanks